src/components/Header/index.jsx

```diff
@@ -39,6 +39,7 @@ function Header({ schema, sourceMode, toggleMode }) {
           className={classes.button}
           color="inherit"
           size="small"
+          variant="outlined" 

```

Comment:
After a second thought, let's do color secondary and remove the variant:

```jsx
<Button
  className={classes.button}
  color="secondary"
  size="small"
  onClick={toggleMode}>
  {sourceMode ? 'hide' : 'show'} source
</Button>
```

Reply:
Applied changes!